sand/extension_interface/search.py

```diff
@@ -1,23 +1,29 @@
 from abc import ABC, abstractmethod
 from typing import Dict
+from sand.models.search import SearchPayload
 
 
-class ISearch(ABC):
-    """ Search Interface to support searches from multiple
+class IEntitySearch(ABC):
+    """ Entity Search Interface to support searches from multiple
         KG datastores.
     """
-
     @abstractmethod
-    def find_class_by_name(self, search_text: str) -> Dict:
-        """Search Class using name"""
+    def find_entity_by_name(self, search_text: str) -> SearchPayload:
```

Comment:
why don't you return a list of items instead?

Reply:
updated the return type